package sect10_TratamentoDeErros;

public class ChecadaVsNaoChecada {
	
	public static void main(String[] args) {
		
		geraErro1();
		geraErro2();
		
	}
	
	static void geraErro1() {
		new RuntimeException("Ocorreu um erro #1");
		
	}
	
	static void geraErro2() {
		new Exception("Ocorreu um erro #2");
		
	}

}
